README.md

```diff
@@ -1,21 +1,21 @@
-[![Build status](https://github.com/git/git/workflows/CI/PR/badge.svg)](https://github.com/git/git/actions?query=branch%3Amaster+event%3Apush)
+[![Build status](https://github.com/git/git/workflows/CI/PR/badge.svg)](https://github.com/git/git/actions? query=branch%3Amaster+event%3Apush)
 
 Git - fast, scalable, distributed revision control system
 =========================================================
 
 Git is a fast, scalable, distributed revision control system with an
-unusually rich command set that provides both high-level operations
-and full access to internals.
+unusually rich command set that provides both high-level operations and low-level operations
+and full access to the internals.
 
-Git is an Open Source project covered by the GNU General Public
+Git is an Open Source project covered by the GNU General Public.
```

Comment:
This full stop is incorrect, as it would end the sentence in the middle of the term "GNU General Public License".